cmd/coreos-assembler.go

```diff
@@ -0,0 +1,170 @@
+package main
```

Comment:
Might be nice to have at least a sentence or two description up here at the top of this file.